common/src/main/java/com/netflix/conductor/common/utils/EnvUtils.java

```diff
@@ -45,4 +48,11 @@ public static String getSystemParametersValue(String sysParam, String taskId) {
         }
         return value;
     }
+
+    public static String getSystemEnv(String key) {
+        if (key == null) {
+            return null;
+        }
+        return System.getenv(key);
```

Comment:
I think instead of getting this as environment variables which need to be passed thru bender scripts, we can make this property variable passed thru config file. This way we can update the property file to pass the new URL instead of bender scripts.

Reply:
done